### PR TITLE
Add Affinity Designer Cheat Sheet

### PR DESCRIPTION
**Pull request by LabLayers**
### What does your Instant Answer do?

Display keyboard shortcuts for Affinity Designer :sparkles: 

### What is the data source for your Instant Answer? (Provide a link if possible)

Affinity Designer's homepage, last slide on the slideshow towards the bottom of https://affinity.serif.com/en-us/designer/ :+1: 

image

In addition to the Mac menubar (examples:)

screen shot 2015-07-11 at 1 06 20 pm

screen shot 2015-07-11 at 1 24 53 pm

screen shot 2015-07-11 at 1 32 06 pm

### Why did you choose this data source?

Official data source :zap: 

### Are there any other alternative (better) data sources?

Zero. :see_no_evil: Zip. :hear_no_evil: Nada. :speak_no_evil: 

### Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)

Vector artists using Affinity Designer.

### Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?

No. :goat: 

### Are you having any problems? Do you need our help with anything?

image

### Where did you hear about DuckDuckHack? (For first time contributors)

About a week ago I heard about this DuckDuckHack meetup ;)

### What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)

image

image

-----
IA Page: https://duck.co/ia/view/affinity_designer_cheat_sheet

### CODE REVIEW

**Comment by moollaza** [comment]
@LabLayers sorry for the delay! Thanks a lot for contributing :+1:

File: share/goodie/cheat_sheets/affinity-designer.json [hunk 1]
```diff
@@ -0,0 +1,433 @@
+{
+    "id": "affinity_designer_cheat_sheet", 
+
+    "name": "Affinity Designer",
+
+    "description": "Vector Editor", 
+    
+    "metadata": { 
+        "sourceName": "Affinity Designer Cheatsheet",
+        "sourceUrl": "https://i.imgur.com/qY4k7xX.jpg"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/affinity-designer.json]: Any chance there's a textual source we can send them to?

**Comment by moollaza** [comment]
LGTM :+1:

File: share/goodie/cheat_sheets/affinity-designer.json (hunk 1 above)
Comment (LabLayers) [inline share/goodie/cheat_sheets/affinity-designer.json]: We an send them to http://www.miguelboto.com/affinity/, but it's not really a cheat sheet but rather a guide of useful videos. Other than that, there's no other source I'm aware of that lists the keyboard shortcuts.

**Comment by LabLayers** [comment]
I find it funny how this PR is the # 1 result on DuckDuckGo:

image

**Pull request merged**